terminal/term.js

```diff
@@ -123,8 +123,10 @@ function addListeners() {
             handleInput($("#input").html());
             inputIndex = 0;
         } else if (key === 38) { //up arrow
-            document.getElementById("input").innerHTML = lastInputs[inputIndex];
-            inputIndex < lastInputs.length-1 ? inputIndex++ : true;
+            if (lastInputs[inputIndex] != undefined) {
```

Comment:
could you just check with a simple boolean `if (lastInputs[inputIndex])` instead? or are you specifically checking for undefined here

Reply:
It needs to check for undefined because if not, undefined will pop up if there is nothing in the command history.

Reply:
I see, that makes sense.